tests/brevitas_examples/test_llm.py

```diff
@@ -200,18 +200,18 @@ def test_small_models_toggle_run_args_pt_ge_2_4(
         "llama",
         "mistral",],
     params=[
-        {
-            "model": "hf-internal-testing/tiny-random-MistralForCausalLM",
-            "act_equalization": "layerwise",
-            "gptq": True,
-            "float_ppl": 31274.05078125,
-            "quant_ppl": 33139.23046875},
         {
             "model": "hf-internal-testing/tiny-random-LlamaForCausalLM",
             "act_equalization": "fx",
             "bias_corr": True,
-            "float_ppl": 33239.5,
-            "quant_ppl": 33283.75390625},])
+            "float_ppl": 33312.0,  # 33239.5,
+            "quant_ppl": 33056.0},  # 33283.75390625},
+        {
+            "model": "hf-internal-testing/tiny-random-MistralForCausalLM",
+            "act_equalization": "layerwise",
+            "gptq": True,
+            "float_ppl": 31056.0,  # 31274.05078125
+            "quant_ppl": 33056.0},])  # 33139.23046875},])
```

Comment:
What do the numbers in the comments mean?

Reply:
Those were the numbers for transformers < 4.46.0. They need to be removed, but I wanted to get confirmation on the preferred way to go regarding the transformers version.

Reply:
Thanks - can you add that to the comment?

Reply:
Maybe we can have a dict that pulls out different reference accuracy based on transformer version? It might get annoying in the long run to keep track of everything, but it could also help catch regression that we might have initially attributed to "numbers have changed"